jobs/frontend/senior-frontend-developer.md

```diff
@@ -0,0 +1,58 @@
+# Senior Frontend Developer
+
+We're looking for a pro-active senior-level frontend developer, who not only solves tasks but suggests improvements on the codebase level as well as on product and company level, and who shares knowledge within the team on meetups and in chats. <b> We are open to a person with ambitions to fill in the Team Lead role. </b>
+
+## About the job
+
+- Salary: up to 5000USD, gross per month.
+- You will work with a team of developers (3-5 people) on the current project developing new features.
+- Usually, our projects last for about 1.5-2 years (but it could be more/less). Our developers don't work on more than 1 project at a time.
+- For development we usually use Kanban or similar process in Trello, Pivotal or Jira, communicate in Slack channels and via Zoom calls, and store documentation in Notion. We also use Holacracy for organizational governance.
+- Apart from working on projects we will encourage you to gain and share knowledge, discuss your personal development and contribute to company's activities.
+- You'll have colleagues all around the globe, variety of channels in Slack like #animal-therapy, #cooking, #ihateremotework and remote team spirit.
+
+## We expect that you
```

Comment:
[Optional]

I think having a certain list of technologies we use might increase a chance of a person applying, makes it clear what to expect and also makes it easier to search. For example, for front-end, we use React almost on all the projects.

It might be just a section `Our tech stack` or `Technologies we use` which describes that our stack depends on a project but usually we use these technologies `<list of technologies>`.

Here is a message from me that highlights some technologies https://datarockets.slack.com/archives/C0H4KK3B7/p1647242319770709.

Also here is a thread where a tech stack is described for each project https://datarockets.slack.com/archives/C50JWS3K5/p1646895843692609

Reply:
@lesha1201 I like your suggestion but I'd merge this PR without it cause this will take some time and then it will require another round of review. However, I don't want to block @dashandrina from being able to present this vacancy to candidates. Can you please add this section in another PR after this one is merged?

Reply:
@atatarenko I will add it.